### DISCUSSION TITLE
Thumb generation for WebP?

### DISCUSSION THREAD

**Discussion by OswinWenlock** [pocketbase/pocketbase#2647] [Q&A] [answered]
Well met,

I've been trying to get the automated thumb generation working with a `.webp` file, but to no luck. It works without error using a `.jpg`, so I should just like to know if it's limited to only `.png` and `.jpg` files or if I'm making a mistake.

I did read in the documentation:
> If your file field has Thumb sizes option, you can get a thumb of an image file (jpg, png)

but wasn't sure if the file type list was exhaustive. 

Cheers.

**Comment by ganigeorgiev** [comment, marked as answer]
WebP thumbs are not supported at the moment, primarily due to the lack of native Go WebP encoder. You can also check https://github.com/pocketbase/pocketbase/issues/825.

The currently supported thumb formats are `.jpg`, `.png` and _partially_ `.gif` (_only the first frame_).

**Reply by OswinWenlock** [reply]
Thank you for the quick response.
